tests/test_bpo.py

```diff
@@ -405,6 +498,10 @@ def __init__(self, *, getitem=None):
     assert body_data["body"].count("[bpo-123](https://bugs.python.org/issue123)") == 2
     assert body_data["body"].count("[something about bpo-123](https://bugs.python.org/issue123)") == 1
     assert "123456" in gh.patch_url
+    if action == 'opened':
```

Comment:
If this is necessary then it might be better to have a helper function for the tests which does everything that's the same and then break this into two separate tests that do the appropriate mock check that's unique to each.